### ISSUE TITLE
connection refused after change_node_status

### ISSUE THREAD

**Issue by PawelPodkalicki** [adshares/ads#73]
I received `Connection refused` error after setting node status. 

Test case is following: dev network 20x20 (genesis attached to full log), 7 vip accounts. 
There are few loops. In each loop every vip administrator changes one bit in own status twice.
First node changes first bit, second node changes second and so on. 
In next loop first node changes bit next to bit changed by last node.

Error is reproducible and occurs always after setting 16th bit (counting from 1) by vip node2:
`(echo '{"run":"get_me"}';echo '{"run":"set_node_status", "node":"2", "status":"32768"}')`

Sending only this request doesn't trigger the error. Error is triggered somehow by set all previous calls.

Full log is available in *20180612_1636_logs_conn_refused_set_node*

**Comment by PawelPodkalicki** [comment]
Node2 still stops in b7c3e67a321fb9fe0b3419a34cd0ae2d80aba110. Error in node2 log:
```
[[1529394112] STOPing validation to start block
[1529394112] STOPed validation to start block
[1529394112] SIGNATURE, get signatures in blk/5B2/8B380/signatures.ok
[1529394112] ERROR, not enough signatures collected (1<3 && 1<3) for 5B28B380
[1529394112] 
SIGNAL: 3
```

**Comment by PawelPodkalicki** [comment]
Fixed in 8ed20cfbd9443459c41de177e2dc0ded1acb97c7
